fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -425,9 +426,10 @@ private static void checkReplicationNum(short replicationNum) throws AnalysisExc
             }
         } else {
             if (replicationNum > backendIds.size()) {
-                throw new AnalysisException("Replication num should be less than the number of available BE nodes. "
-                        + "Replication num is " + replicationNum + " available BE nodes is " + backendIds.size() +
-                        ", You can change this default by setting the replication_num table properties.");
+                throw new AnalysisException("Table replication num should be less than " +
+                        "of equal to the number of available BE nodes. "
```

Comment:
1. "than of equal"  --> "than or equal"
2. print the replicationNum  (you removed it)
3. "You can change this default by ..."  -->  "You can change it be ..."
    that, 'default' here is ambiguous.